parachain/primitives/core/src/outbound.rs

```diff
@@ -6,6 +6,16 @@ use sp_std::{borrow::ToOwned, vec, vec::Vec};
 
 pub type MessageHash = H256;
 
+/// Priority for submit the message ticket to OutboundQueue
+#[derive(Copy, Clone, Encode, Decode, PartialEq, Eq, RuntimeDebug, TypeInfo)]
+pub enum Priority {
```

Comment:
I've realised that we can't use this `Priority` type any more. See my comment below, for why we need to use message origin as the priority.

Reply:
https://github.com/Snowfork/snowbridge/pull/949/commits/239f3c37a261c7bfda76fb6dcfbce5805eeecee4